### PR TITLE
Add Trino 374 release notes

### LINKED ISSUE #11402: Release notes for 374

### PR DESCRIPTION
**Pull request by mosabua**
## Description

Assemble the release notes for the upcoming Trino 374 release

> Is this change a fix, improvement, new feature, refactoring, or other?

Improvement.

> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)

Documentation only.

> How would you describe this change to a non-technical end user or system administrator?

Add the release notes for Trino 34  with sections for general, security, connectors and other topics as included in the release. 

## Related issues, pull requests, and links

* Fixes #11402 
* Follow up to  #11290

## Documentation

( ) No documentation is needed.
(✅ ) Sufficient documentation is included in this PR.
( ) Documentation PR is available with #prnumber.
( ) Documentation issue #issuenumber is filed, and can be handled later.

## Release notes

(✅ ) No release notes entries required.
( ) Release notes entries required with the following suggested text:

##  Verification for each commit

Format: commit, committer/maintainer, PR/issue number, ✅ / ❌  RN, ✅ / ❌   docs
✅  RN - release note added and verified, or assessed to be not necessary, set to ❌ RN before completion
✅  docs - need for docs assessed and merge, or assessed to be not necessary, set to ❌ docs before completion

## 9 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/b123e5c12081a52cec7046de4fc6e335de848a99 @ebyhr #11382  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/3ed8e67cc2d45a4d079596d88477edcaf5a58dea @findepi #11086 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/808b677b510552c1f8808e826442aad13b81f03e @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/812f79bb703215ce10e5bdd581a75a1f82ad5532 @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2e2ec687c726feccb8462a19abfcb8e409e8d27d @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/cd101d7563418eeed285e0ff80fc3fe70fd2d2fd @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/e3f1a317845a69f7942401b404143c47ae59553b @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/8686287f08cb505b2c8df0d61fc517fe563726f7 @findepi #11086  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6b6c938a904276b0c2c13f164c7ba16ea22a180b @findepi #11086  ✅ rn ✅ docs

## 10 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/70dd9789dd43e55122cd1ef498ec8cfc8157dec1 @sopel39 #11392  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/d0701f28a56717f5ae2c5c0525942550ee9062e1 @hashhar #11399 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2ce6cf4dd6fad67040d75614403586bdef1aaa0c @hashhar #11399 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2cabc97947c603ffdfa62bc6def831dfb138d461 @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/64d8daad0ea32268244b6d986e82195c73593bee @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/47b77a2f42248a9c9622394e437fbe4914300c7b @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/4ba555e2e2461c62767799f2d2824c3ecf8588f6 @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/9d3783dc5b4a79a9fb9659aed14602bb1dee7fe4 @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/abb4c035bd6fe9c580178133c56fc5335525c4d0 @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/98c2500f8d90a2a8fa1ccc06433c3ceef1f3387e @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/ef8aaa92f37680a3cc2ab7b24ee1e72ba1b85cef @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/09521ea19d8f8565a4fb23daa5d383867dd0895c @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2988194cf35089e4b9f00f620871bcd886ff4031 @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/99ef4ec162b3b099762e4d6727ee3db641e1445c @losipiuk #10432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/f551dd2ccfd7b7d5d55d273983bae93949e0b840 @martint #11414 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/426fd5fc90fc2033c609834e5702b5ef6660d358 @ebyhr #11409 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/4ba98593053ee0b0dabc01f018b3ce5abb953d39 @ebyhr #11412 ✅ rn ✅ docs

## 11 Mar 2022  ✅ 

* https://github.com/trinodb/trino/commit/af86809e937e0cc0a63effc6dbe22a8966ca4ae9 @findepi  #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/50f9cfee9950d6c7cf73c1aa0cde4a284e5e7737 @findepi   #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/fe16de139b3c077f987feb8dfca50fb3e7bd45d4 @findepi   #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/a8f1d675807788cd2468fb5d7a3ddc1108a6e084 @findepi   #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/70c988b7013dfed259b14c8b871e9a0bc4f133c4 @findepi   #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/e5732f8857f8e0a017fe015eb2fe12673a5e49b4 @findepi   #11083 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/74661d64ab5796662d3cdad8391fb64a6b4bee58 @kokosing #11014 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/fdad162cf9a4d0b93bb7970ec959a0d0aabdf9bd @ebyhr #11421 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/271a713651dc42bd25b74f0fb2b1b804f41ce15d @ebyhr #11424  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/637d235047a43260c510a4a24b0f78062a9d1c37 @findepi #11418  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/79faeb07737ffd935032ffdcc40010b10d75f50d @sopel39 #11011  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/3f6ae4223f0ecc8ead68e2f4c5447df6da6ea1cd @sopel39 #11011  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/c9ca00755ad22e78f65385cdf196faaf0865f6ae @sopel39 #11011  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/c950e303524cb1bd9ebae5a7234b1f11773db9f3 @sopel39 #11011  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/80638414be40cae56f456609f5dc4243630a1e04 @sopel39 #11324  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/291b65dd831b47e9596a67ce3fac99cbe5ed35ae @sopel39 #11324  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/67d2df9ac15f8169a492dd8e9ccc44fb879424f5 @sopel39 #11324  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/a4c7745f15b21c8b0d86bf2e878b38da021720c8 @sopel39 #11324  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/48d3fe3c9ea14861ca44124dcd4fec92e1d9bf78 @sopel39 #11324  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/a1f2eeea33f4e188f2a1deed525991dd4d70244d @sopel39 #11426 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5297372129b5a4f783999cfd8ca6f3e872022cd4 @sopel39 #11432 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/396d24963fd4059821927d87e9bb75fcbd932c2c @hashhar #3602  ✅ rn ✅   docs, covered in https://github.com/trinodb/trino/pull/11187
* https://github.com/trinodb/trino/commit/6a70525addde6ae39419e6c4348c88950597bdb7 @hashhar #11205 ✅ rn ✅   docs
* https://github.com/trinodb/trino/commit/eed07af1d3272ded8552d6a528e35b265e7c0dbf @ebyhr  #11425 ✅ rn ✅ docs

## 12 Mar 2022 ✅ 

* no commits

## 13 Mar 2022  ✅ 

* https://github.com/trinodb/trino/commit/6c53b97f79fbe8ed9b6f53b27ca3911ea796f8fa @hashhar #11433 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/bc0c8294c5246615e12c98b08e59899ad70abede @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/146d9c0ce4d4a5ab52e454b4f2cf20e170cdad66 @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/ec28a572bac037d1ebdd845da40a13de5e536d06 @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/0ec20cce411447400f9955bf043e72043a9f6d54 @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5d4995b4177ecc20254d6bbafe72a61ca398aa89 @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/65559c53263316483718c42e92d3b3b54aff0ce5 @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/fd762193981122750444c00734f919027af3fd1e @dain #10378 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/736c160dffe97544612b524802631e6b63ecf596 @ebyhr #11429 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2057d04c65b4d4318ca9954baa3a1a11d6da7fb1 @Praveen2112 #11366 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/78eca2aeb507f16e1cd675a551d70480303af7a0 @hashhar #11449 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/efb7ad831c328fc9db5a130f8eb5940922b1d5be @hashhar #11451 ✅ rn ✅ docs

## 14 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/11c9b81234e5ff63454897414ce98d2baecf1371 @hashhar #11301  ✅  rn ✅ docs , https://github.com/trinodb/trino/pull/11474
* https://github.com/trinodb/trino/commit/a29f4724be33e8a69c63279444b5d38d3dffa549 @findepi #11438 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2594f826b5aae2219725399503492611121f8b7d @sopel39 #11398 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/7b3bed7329d9f9d3f7d668ca11c2e800c41bece4 @sopel39 #11398 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5dd5d4d1d95f3a8cbca8d4288acb709276868879 @sopel39 #11398 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/f4d5751d9288e2381fd287abdbe0b03922bcbbff @kokosing #11445 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/e6bc5f4ecc4d8246cf3870beb2fac0db1d1e1be0 @findepi #11465 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/9dab628d86024f62843a281ee664bc4ef6cd3a0c @arhimondr #11235 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/d7f4d1db66b5d44a21e1afb7dfff432ea2941174 @arhimondr #11235 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6476b48b9411391dda75ce3e7e1210aa3a4184d8 @hashhar #11387 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/a549915f4f30abb137bc1040f7fc39b5a3e5f2b7 @hashhar #11387 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/21d4224116036f4b55164b0d81a1c9789b3ad478 @hashhar #11387 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/211c5944e77f5ee0bef3bac713ee8ecf49484cfd @hashhar #11462 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6c0a80e218e56a6c4e61c15d0ab22984d4a41c8a @dain #11358 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/1abc7eb9d17fd2a795b9c25b0e8dc1dc96e68b99 @dain #11358 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2bfd417de5eec4a88fc1f1a3e4fcfe9a0e751685 @dain #11358 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/90acb84de27ad76bc7125e1f84943d4646753a20 @findepi #11437 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/e84dff8357a0e314db12f915d3cb8aeaf18522a3 @ebyhr #11440 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/b216e2c3605b943b342c0504ede38d47981ca0c8 @ebyhr #11474 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/39607b53fa8f5f410120bea284e1fa453a7b3303 @ebyhr #11459 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/d8e111cf4aa8dd2807f4247c32c5bebb6abad05c @ebyhr  #11453 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/61b76578bf0182435df240d781612113052ce5dc @ebyhr #11457 ✅ rn ✅ docs

## 15 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/d932ed476b9e3e4d996f843faedb8d125aa2a074, @ebyhr  #11481 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/9089794774cfe7e599840a8f8a1622953889093a @losipiuk #11480 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/20df89aa5bbc13d4249b36d99a2368110a1db772 @losipiuk #11419 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6e0a16032046b463bcc8c768fc191a612354cb66 @losipiuk #11419 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/9617a95e1404ab4ed62b4a256dde7a02bee49919 @losipiuk #11419 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/eb61a7ae35150c03cc388fceea69364ef8cb8211 @losipiuk #11419 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/4e6207243f654c1ba7d42f04c76ba47e242022e5 @findepi #11473 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/95cabd6ef46ec71c142eb913731c82527bd0b4a2 @findepi #11125 ✅  rn ✅  docs
* https://github.com/trinodb/trino/commit/a2ffa7544c5311235a78dab3843f9ece4ee12689 @findepi #11125 ✅  rn ✅  docs
* https://github.com/trinodb/trino/commit/b44824b1388588baa682f2438b93ddda13c53783 @findepi #11125 ✅  rn ✅  docs
* https://github.com/trinodb/trino/commit/892523d954519c405fe8c0d8c65ce39bae07d939 @hashhar #11478 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/8095a023e4050f7dd7ceeb4c75eb109edd80eac7 @findepi #11472 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/4976d7931ffe633f9b9d79661ed6ed90a7e9a085 @ebyhr #11487 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/0d1e5cc986307d3ae61b07cbc328c6235b9c64b3 @ebyhr #11487 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/d5e532b15fce0cdb3905aae77838659391b781d5 @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/7abed73671cca02634393eaa694e6a48cfa54f8a @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/c9ad95d2889e056b41002dc69226a4d273fbd7d1 @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/251688495048a1fefda1656e24335d0598d3238d @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6d0829a5100273150433b5012e22d650595ae9e1 @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/724bf9727f834a94b013a76e95f96c556ede1175 @findepi #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/1c58a3a6cbaf4dd2194f7bf57c4c318593dcc062 @findepi  #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/582b2d877c989547d26221a81f216e57f3652871 @findepi  #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/beaadc82413d438cc26baa848c19af78d7540e23 @findepi  #10845 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/29b4846e3f5777170e7c0de7631ed2d724e6a610 @electrum #10625  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/48e0bfa4fccf0ca789b6302185ef01865362d0a9 @electrum #10625  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/349b3ce2d433b42fc468d3ab81cd119f30b26f92 @arhimondr #11479 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/a690d4ad4f1b40e5103918c85732496b5bef3b59 @hashhar  #11502 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5187a3d4785a049f0c80fa7287f633a15bc306fb @dain #11443  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/876f3af49082134f527e8b526f7a04400318b126 @dain #11443  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/b79d485c8ba0feb1458c1b2eb97fba6659394cda @dain #11443  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/359568d0690fed99ec5f47a97117ac14c64e4e27 @hashhar #10411  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/665e87cb76b6c98e0f2c1c1e52023d54263b4da7 @hashhar  #10411  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/e0cf4a9e8e33403619748ccbe18dce49911d3d3b @hashhar #11316   ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/1170bd9ccff5b163c8325d6a2e107954217d3947 @hashhar #11446  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/c8621304a41a2735592eed1791e583bcf345dc23 @hashhar #11446  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/64686a84cd747aec8f3d47c47c83a06cff83994b @hashhar #11446  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5e6b8e55b93e1cd8e824ce5e8c61692e94ec3fb1 @hashhar #11500  ✅ rn ✅ docs

## 16 Mar 2022 ✅ 

* https://github.com/trinodb/trino/commit/1cfa4f618954a33b7a1bbbc06f25ef20c7c3c53e @hashhar #11495  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/8add1e79757dfb583866e01cd88607cdde52e402 @findepi #11470  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/b704cee81ff458e724de566e7cb34e1a74681ae8 @findepi #11464 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/5b5b8c64047a8d4d9c2c6064eee29881610e8675 @findepi #11464  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/b1710ed03cd2b0f43991444c67c4808eae7d0831 @findepi #11496 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/98ded660694bb8f39fe94ee59175e34d8cde7235 @findepi #11496 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/ae3d11dc258c099e8293b39da456bcadd0491d83 @findepi #11501 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/0d4dd15efbe1750f207a5b3665ae8d22af99d35c @findepi  #11501 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/c8ebfd7c2f84f160db9be0d86283e575b7966746 @findepi  #11501 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/784d3d15e4b765d24d382d4f1252f3c789e9b361 @findepi #11504 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/b1337f8479527c5aca5ef7a6d8121782d01bc51e @hashhar #11482 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/820680248c16c7733f5482597107a2a19a9b0ccf @hashhar #11482 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/010e8407f317aae36c87c7bcd4ad1a0d5514ae81 @losipiuk #10169 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/504ba97dbeb02c928720564e5a306dd5126560aa @losipiuk #11497 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/63e0696b257cadef5a25bcb2eecae7ac7e75e79f @ebyhr #11490 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/f82977720f18fd6dce9959364d142e2a6710f9ea @ebyhr #11490 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/51ea8dd496988cbe432e20b31366c1a51b056407 @ebyhr #11490 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/6d1d817379127e6b05de1bb6f007d279f698eadc @ebyhr #11490 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/0777c46a192848d5318bf530bcb2d235cab3edcf @ebyhr #11490 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/21ffb097dcf45275f87669117cd55c38d0471a31 @kokosing #11489 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/4b2633d9f03c9957f9385a3146417eb1aaf5ac98 @kokosing #11489 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/84d17d3badd34253d1a52476221e743fbe505dc6 @kokosing #11489 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/2605f0cb83a459af47b751d0b8ce3df09029836a @kokosing #11489 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/00cc00e41794eb07ac3c3691a13591fad19facf4 @arhimondr #11452 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/01424e58491b8d2b3cb75cea9f1f66de5cf94eaf @arhimondr #11174 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/687ea115eddc8673e80230092c9662c0ca2aaa53 @arhimondr #11174 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/758145f62f9f5f1d19cee484f5d5c6d38e6891a9 @arhimondr #11174 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/20ef21709e0c6dc973df4d3814f8c2faabf7fc22 @arhimondr #11174 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/f4f1496145e77e15f2940669cf340ae1d2b16e4f @arhimondr #11174 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/ad27b121d78abf24980b96aadc2521288484f448 @losipiuk #11404 ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/43f702f2fac689e2d283877d2ab207b34807e3b1 @losipiuk #11509  ✅ rn ✅ docs
* https://github.com/trinodb/trino/commit/8b93d7794345f669f8b8127f3d7cd8a82742cc26 @losipiuk #11519  ✅ rn ✅ docs

### CODE REVIEW

File: docs/src/main/sphinx/release/release-374.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improved handling for queries where individual tasks require lots of memory.
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-374.md:6]: @losipiuk @martint  .. could we cut this shorter to only be 

Improve retry handling for queries where individual tasks require lots of memory. 


The rest feels like documentation and @losipiuk and @jhlodin are working on this now to get ready soon. Ideally a first version even with the 374 release..

Comment (jhlodin) [reply docs/src/main/sphinx/release/release-374.md:7]: +1, this behavior is covered in https://github.com/trinodb/trino/pull/11404

Could reword to "Expanded :doc:`/admin/fault-tolerant-execution` to improve handling for queries where individual tasks require lots of memory" as a way to sneak a link in there.

File: docs/src/main/sphinx/release/release-374.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improve performance of `GROUP BY` with a large number of groups. ({issue}`11011`)
+* Improved handling for queries where individual tasks require lots of memory.
+  If the task fails due to out-of-memory error it is retried with a full node at
+  its disposition. Mechanism is only active if task based retries are enabled
+  with `retry-policy` is set to `TASK`. ({issue}`10432`)
+* Produce better query plans by improving cost-based-optimizer estimates in the
+  presence of correlated columns. ({issue}`11324`)
+* Allocate a new Slice instead of returning a view when decoding
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-374.md:13]: @losipiuk @linzebing @arhimondr .. can we cut this down to be a shorter somehow?

File: docs/src/main/sphinx/release/release-374.md (hunk 1 above)
Comment (martint) [reply docs/src/main/sphinx/release/release-374.md:7]: I would rephrase along the lines of:

> Improve handling of queries where individual tasks require lots of memory when `retry-policy` is set to `TASK`

File: docs/src/main/sphinx/release/release-374.md (hunk 2 above)
Comment (linzebing) [reply docs/src/main/sphinx/release/release-374.md:13]: Maybe the following two bullets:
- Reduced unnecessary memory copying when consuming data from exchanges.
- Fixed memory accounting of pages involving `VariableWidthBlocks`s

Comment (martint) [reply docs/src/main/sphinx/release/release-374.md:13]: > Fixed memory accounting of pages involving `VariableWidthBlocks`

VariableWidthBlock is an implementation detail that users won't understand. It should be described in higher level terms, such as:

* Fix memory accounting for queries involving certain variable-width data types such as `varchar`, `varbinary`, etc.

Comment (mosabua) [reply docs/src/main/sphinx/release/release-374.md:13]: Updated

File: docs/src/main/sphinx/release/release-374.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Implement parallel read for S3-based spooling in [fault-toleration execution](/admin/fault-tolerant-execution). ({issue}`11050`)
```
[the remark this answers is not in the document]
Comment (jhlodin) [reply docs/src/main/sphinx/release/release-374.md:5]: Should be *fault-tolerant

File: docs/src/main/sphinx/release/release-374.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+# Release 374 (16 Mar 2022)
+
+## General
+
+* Implement parallel read for S3-based spooling in
```
Comment (martint) [inline docs/src/main/sphinx/release/release-374.md:5]: Improve performance when reading from S3-based spool for fault tolerant execution.

File: docs/src/main/sphinx/release/release-374.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+  [fault-toleration execution](/admin/fault-tolerant-execution). ({issue}`11050`)
+* Improve performance of queries with `GROUP BY` clauses. ({issue}`11392`)
+* Improve performance of `GROUP BY` with a large number of groups. ({issue}`11011`)
+* Improve handling of queries where individual tasks require lots of memory when
+  `retry-policy` is set to `TASK`. ({issue}`10432`)
+* Produce better query plans by improving cost-based-optimizer estimates in the
+  presence of correlated columns. ({issue}`11324`)
+* Fix memory accounting for queries involving certain variable-width data types
+  such as `varchar` or `varbinary`., Benchmark shows a 0.7%-2.7% CPU efficiency
```
Comment (martint) [inline docs/src/main/sphinx/release/release-374.md:14]: Remove extra `,`

Comment (martint) [reply docs/src/main/sphinx/release/release-374.md:14]: Also, the mention about benchmarks seems disconnected from the first part, which describes a memory accounting issue. I would just remove that section, and maybe add "Fix memory accounting and improve performance for queries ..." at the front.

File: docs/src/main/sphinx/release/release-374.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+* Support storing a comment when adding new columns. ({issue}`11487`)
+
+## PostgreSQL connector
+
+* Improve performance of queries involving `OR` with simple comparisons and
+  `LIKE` predicates by pushing predicate computation to the PostgreSQL database.
+  ({issue}`11086`)
+* Improve performance of aggregation queries with certain complex predicates by
+  computing predicates and aggregations within PostgreSQL.
+  ({issue}`11083`)
```
Comment (martint) [inline docs/src/main/sphinx/release/release-374.md:72]: Move this to previous line
